boards/lilygo/ttgo_toiplus/doc/index.rst

```diff
@@ -0,0 +1,199 @@
+.. zephyr:board:: ttgo_toiplus
+
+Overview
+********
+
+Lilygo TTGO T-OI-PLUS is an mini IoT development board based on
+Espressif's ESP32-C3 WiFi/Bluetooth dual-mode chip.
+
+It features the following integrated components:
+
+- ESP32-C3 SoC (RISC-V 160MHz single core, 400KB SRAM, Wi-Fi, Bluetooth)
+- on board Grove connector
```

Comment:
Would be nice (but not mandatory) to describe it in the devicetree

Reply:
Got it, might save it for a future PR but not this one